recipe/meta.yaml

```diff
@@ -13,13 +13,14 @@ source:
 
 build:
   number: 0
+  # numcodecs is not currently supported on s390x
+  skip: True  # [py<37 or s390x]
   script:
     - export DISABLE_NUMCODECS_AVX2=""  # [unix]
     - export DISABLE_NUMCODECS_SSE2=""  # [arm64]
     - set DISABLE_NUMCODECS_AVX2=""  # [win]
+    - export CFLAGS="${CFLAGS} -pthread"  # [aarch64 or ppc64le]
```

Comment:
Just wondering if there's a reasoning for this or if it has just been taken from conda-forge?

Reply:
Yeah, it has been taken from conda-forge https://github.com/conda-forge/numcodecs-feedstock/commit/0e0a49dffc80d4e19f0f6af52d8ef1cbbc43696c

Reply:
hmm, not sure about putting in stuff that we don't understand. did it work without this?